examples/bsoncxx/builder_stream.cpp

```diff
@@ -58,4 +58,17 @@ int main(int, char**) {
     using builder::stream::finalize;
     auto myQuery = document{} << "foo"
                               << "bar" << finalize;
+
+    // There is a special concatenate_doc helper to add all keys and corresponding values from one
+    // document into another.
+    using bsoncxx::builder::concatenate_doc;
+    doc << concatenate_doc{myQuery.view()};  // `doc` now has "foo": "bar".
```

Comment:
Maybe the comment would be better if it showed the actual document. "now has" could mean a few different things.